docs/docs/multi-dc.md

```diff
@@ -0,0 +1,176 @@
+Multi Datacenter
+================
+
+As of version 0.2.0 Cassandra on Mesos allows for providing configuration
+to allow multiple datacenters to be aware of one another.
+
+
+## How it works
```

Comment:
It'd be really useful to add a high level explanation of why multiple datacenter support is useful/interesting. (You could also link to: http://www.datastax.com/dev/blog/deploying-cassandra-across-multiple-data-centers)